src/Language/DifferentialDatalog/Module.hs

```diff
@@ -64,28 +64,31 @@ data DatalogModule = DatalogModule {
     moduleDefs :: DatalogProgram
 }
 
--- standard library module name
-stdname :: ModuleName
-stdname = ModuleName ["std"]
+-- Standard library module name.
+stdLibs :: [ModuleName]
+stdLibs = [ModuleName ["std"], ModuleName ["internment"]]
```

Comment:
this should probably be documented too, because users need to know this.

Reply:
Documentation is on its way...